tensorboard/webapp/widgets/card_fob/card_fob_types.ts

```diff
@@ -88,3 +88,14 @@ export interface CardFobGetStepFromPositionHelper {
    */
   getStepLowerThanAxisPosition(position: number): number;
 }
+
+/**
+ * Contents of the timeSelectionChanged event.
+ */
+export type TimeSelectionChanged = {
+  // Affordance for internal analytics purpose. When no affordance is specified or is
+  // undefined we do not want to log an analytics event.
+  affordance?: TimeSelectionAffordance;
+  incomplete?: boolean;
```

Comment:
Is this really the best attribute name?
It is used to indicate the user is still dragging the fob and thus the "Change" is not complete.